### PR TITLE
Initial structure and /orc endpoint

### PR DESCRIPTION
**Pull request by marylin-moya**

### CODE REVIEW

File: src/main/java/com/jalasoft/webservice/utils/FileManager.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
+package com.jalasoft.webservice.utils;
+
+/*
+ *
+ *  Copyright (c) 2019 Jalasoft.
```
Comment (pradovc01) [inline src/main/java/com/jalasoft/webservice/utils/FileManager.java:3]: Please, could you add the following comments: class name, version information, date, and copyright notice (it is detailed at  https://www.oracle.com/technetwork/java/javase/documentation/codeconventions-141855.html -section 3. File Organization )

File: src/main/java/com/jalasoft/webservice/Constants.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
+package com.jalasoft.webservice;
+
+/*
+ *
+ *  Copyright (c) 2019 Jalasoft.
```
Comment (pradovc01) [inline src/main/java/com/jalasoft/webservice/Constants.java]: Please move Copyright information at first line

File: src/main/java/com/jalasoft/webservice/Constants.java [hunk 3]
```diff
@@ -0,0 +1,16 @@
+package com.jalasoft.webservice;
+
```
Comment (pradovc01) [inline src/main/java/com/jalasoft/webservice/Constants.java]: Maybe, this class could be moved to a folder where different utilities or resources classes will be included on it, what do you think?

File: src/main/java/com/jalasoft/webservice/utils/FileManager.java (hunk 1 above)
Comment (marylin-moya) [reply src/main/java/com/jalasoft/webservice/utils/FileManager.java:3]: Paolo give us this Copyright to configure it in our IDE. Also this page has a very old recommendation for Copyright.

File: src/main/java/com/jalasoft/webservice/Constants.java (hunk 2 above)
Comment (marylin-moya) [reply src/main/java/com/jalasoft/webservice/Constants.java]: done

File: src/main/java/com/jalasoft/webservice/Constants.java (hunk 3 above)
Comment (marylin-moya) [reply src/main/java/com/jalasoft/webservice/Constants.java]: done

**Pull request merged**